extensions/interactions/base_test.py

```diff
@@ -150,6 +148,32 @@ def test_interaction_properties(self):
             'default_value': 1,
         }])
 
+    def test_interaction_rules(self):
+        def _check_num_interaction_rules(interaction_id, expected_num):
+            interaction = interaction_registry.Registry.get_interaction_by_id(
+                interaction_id)
+            self.assertEqual(len(interaction.rules), expected_num)
+
+        _check_num_interaction_rules('MultipleChoiceInput', 1)
```

Comment:
I think this is more of a sanity check on the registry and the .rules property than a check on individual interactions, so, leaving as-is.